
geometry_msgs/TransformStamped pose
bool relative

---
bool success